### ISSUE TITLE
'RADIUS_FOR_CISCO_ASA' & 'AWS_FEDERATED_LOGIN' not recognized as valid ApplicationSignOnModes 

### ISSUE THREAD

**Issue by mvitt** [okta/okta-sdk-python#167]
When getting a list of all applications via: `apps, resp, err = await client.list_applications()` it fails with the message: `'RADIUS_FOR_CISCO_ASA' is not a valid ApplicationSignOnMode` & `'AWS_FEDERATED_LOGIN' is not a valid ApplicationSignOnMode` respectively. It seems that those two application sign on modes need to be added to the `application_sign_on_mode.py` file. Also, equivalent models need to be created for each. Once those models are created, they need to be added to the `OKTA_APP_SIGN_ON_TO_MODEL` variable in the `constants.py` file.

**Comment by laura-rodriguez** [comment]
Hi @mvitt,

Thanks for reporting this issue. We'll review this internally and provide you an update when we have more info to share.

**Comment by serhiibuniak-okta** [comment]
@mvitt, thanks again for detailed issue. Models are being generated according to our openapi spec. First, we need to update openapi spec. This requires some time. To unblock you, I can suggest some workarounds. For example, it is possible to get original response body (before it is being mapped to models):
```py
apps, resp, err = await client.list_applications()
orig_resp = resp._body
```
And depending on your needs, you can filter failed apps or process manually or something else. Is it helpful?

**Comment by serhiibuniak-okta** [comment]
@mvitt After some internal discussions, we've decided to avoid errors so far by using "custom" signOnMode when we encounter any "new" signOnMode. Thus, sdk will pick basic Application model for such apps and `list_applications` won't fail. 

Here's PR: https://github.com/okta/okta-sdk-python/pull/168. It will be added within next sdk release.

Please, reach out to us if you have any additional questions or concerns.

**Comment by serhiibuniak-okta** [comment]
@mvitt It turned out, there might be another issues related to signOnMode, thus a new PR should cover it: https://github.com/okta/okta-sdk-python/pull/174
It is merged and will be included into upcoming release. Will keep you posted.

**Comment by serhiibuniak-okta** [comment]
@mvitt SDK v1.4.1 released. I'm closing this issue as resolved. Feel free to reopen this issue if your problem still exists.
